lib/bounding-boxes.ts

```diff
@@ -196,31 +222,214 @@ const countryBoundingBoxData: Record<string, CountryBoundingBox | undefined> = {
   "Saint Lucia": [-61.07, 13.72, -60.85, 14.1],
 };
 
+const alphaTwoCountryCodeBoundingBoxData: Record<
```

Comment:
Made this because I don't want country names changing to break anything anymore :sob:

Ideally, we switch the country stuff to use this one as well and then delete the country one entirely. I'd love to do that and that was kind of my secret motive for adding `alphaTwoCountryCode` on the API but one thing at a time.